From f2545b7189028f9dc7bb97e5d5b78190627cadd7 Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Wed, 11 Mar 2015 20:26:04 +0100
Subject: [PATCH] linker: avoid logcat warning with pre-KK/LP blobs

---
 linker/linker.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index 54867dc..36add53 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -2178,7 +2178,7 @@ bool soinfo::LinkImage(const android_dlextinfo* extinfo) {
   if (has_text_relocations) {
     // Make segments writable to allow text relocations to work properly. We will later call
     // phdr_table_protect_segments() after all of them are applied and all constructors are run.
-    DL_WARN("%s has text relocations. This is wasting memory and prevents "
+    DEBUG("%s has text relocations. This is wasting memory and prevents "
             "security hardening. Please fix.", name);
     if (phdr_table_unprotect_segments(phdr, phnum, load_bias) < 0) {
       DL_ERR("can't unprotect loadable segments for \"%s\": %s",
-- 
2.1.0

