From 482ec213e2ff30bc095ed1a4d564049381a0f9e9 Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Tue, 6 Jan 2015 20:39:16 +0100
Subject: [PATCH] init: add a mechanism to extend the property service in
 device configs

---
 init/property_service.c | 70 +++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 70 insertions(+)

diff --git a/init/property_service.c b/init/property_service.c
index 1902b77..2530fd9 100644
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -56,6 +56,76 @@ static int property_area_inited = 0;
 
 static int property_set_fd = -1;
 
+/* White list of permissions for setting property services. */
+#ifndef PROPERTY_PERMS
+struct {
+    const char *prefix;
+    unsigned int uid;
+    unsigned int gid;
+} property_perms[] = {
+    { "net.rmnet0.", AID_RADIO, 0 },
+    { "net.gprs.", AID_RADIO, 0 },
+    { "net.ppp", AID_RADIO, 0 },
+    { "net.qmi", AID_RADIO, 0 },
+    { "net.lte", AID_RADIO, 0 },
+    { "net.cdma", AID_RADIO, 0 },
+    { "ril.", AID_RADIO, 0 },
+    { "gsm.", AID_RADIO, 0 },
+    { "persist.radio", AID_RADIO, 0 },
+    { "net.dns", AID_RADIO, 0 },
+    { "sys.usb.config", AID_RADIO, 0 },
+    { "net.", AID_SYSTEM, 0 },
+    { "dev.", AID_SYSTEM, 0 },
+    { "runtime.", AID_SYSTEM, 0 },
+    { "hw.", AID_SYSTEM, 0 },
+    { "sys.", AID_SYSTEM, 0 },
+    { "sys.powerctl", AID_SHELL, 0 },
+    { "service.", AID_SYSTEM, 0 },
+    { "wlan.", AID_SYSTEM, 0 },
+    { "gps.", AID_GPS, 0 },
+    { "bluetooth.", AID_BLUETOOTH, 0 },
+    { "dhcp.", AID_SYSTEM, 0 },
+    { "dhcp.", AID_DHCP, 0 },
+    { "debug.", AID_SYSTEM, 0 },
+    { "debug.", AID_SHELL, 0 },
+    { "log.", AID_SHELL, 0 },
+    { "service.adb.root", AID_SHELL, 0 },
+    { "service.adb.tcp.port", AID_SHELL, 0 },
+    { "persist.logd.size",AID_SYSTEM, 0 },
+    { "persist.sys.", AID_SYSTEM, 0 },
+    { "persist.service.", AID_SYSTEM, 0 },
+    { "persist.service.", AID_RADIO, 0 },
+    { "persist.security.", AID_SYSTEM, 0 },
+    { "persist.gps.", AID_GPS, 0 },
+    { "persist.service.bdroid.", AID_BLUETOOTH, 0 },
+    { "selinux." , AID_SYSTEM, 0 },
+    { "net.pdp", AID_RADIO, AID_RADIO },
+    { "wc_transport.", AID_BLUETOOTH, AID_SYSTEM },
+    { "build.fingerprint", AID_SYSTEM, 0 },
+    { "partition." , AID_SYSTEM, 0},
+    { NULL, 0, 0 }
+};
+/* Avoid extending this array. Check device_perms.h */
+#endif
+
+/*
+ * White list of UID that are allowed to start/stop services.
+ * Currently there are no user apps that require.
+ */
+#ifndef CONTROL_PERMS
+struct {
+    const char *service;
+    unsigned int uid;
+    unsigned int gid;
+} control_perms[] = {
+    { "dumpstate",AID_SHELL, AID_LOG },
+    { "ril-daemon",AID_RADIO, AID_RADIO },
+    { "pre-recovery", AID_SYSTEM, AID_SYSTEM },
+    {NULL, 0, 0 }
+};
+/* Avoid extending this array. Check device_perms.h */
+#endif
+
 typedef struct {
     size_t size;
     int fd;
-- 
2.1.0

