From e8247c238b2199a1c2fd8de86474ec3d8fed95fe Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Tue, 6 Jan 2015 13:27:59 +0100
Subject: [PATCH] camera.tegra: fix camera_common.h include

---
 include/hardware/camera_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hardware/camera_common.h b/include/hardware/camera_common.h
index dadbc8f..8837bf6 100644
--- a/include/hardware/camera_common.h
+++ b/include/hardware/camera_common.h
@@ -24,7 +24,7 @@
 #include <sys/types.h>
 #include <cutils/native_handle.h>
 #include <system/camera.h>
-#include <system/camera_vendor_tags.h>
+#include "../../../../system/media/camera/include/system/camera_vendor_tags.h"
 #include <hardware/hardware.h>
 #include <hardware/gralloc.h>
 
-- 
2.1.0

