From d23e502a186f6677246c65ce1a4427e5552460c8 Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Wed, 12 Nov 2014 16:57:25 +0100
Subject: [PATCH 7/7] build: allow build with make 4.0

---
 core/main.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/main.mk b/core/main.mk
index ead6029..7400c00 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -42,6 +42,7 @@ endif
 ifeq (,$(findstring CYGWIN,$(shell uname -sm)))
 ifeq (0,$(shell expr $$(echo $(MAKE_VERSION) | sed "s/[^0-9\.].*//") = 3.81))
 ifeq (0,$(shell expr $$(echo $(MAKE_VERSION) | sed "s/[^0-9\.].*//") = 3.82))
+ifeq (0,$(shell expr $$(echo $(MAKE_VERSION) | sed "s/[^0-9\.].*//") = 4.0))
 $(warning ********************************************************************************)
 $(warning *  You are using version $(MAKE_VERSION) of make.)
 $(warning *  Android can only be built by versions 3.81 and 3.82.)
@@ -51,6 +52,7 @@ $(error stopping)
 endif
 endif
 endif
+endif
 
 # Absolute path of the present working direcotry.
 # This overrides the shell variable $PWD, which does not necessarily points to
-- 
2.1.3
