From 8d47199d1d8034ce4cdd7dce3b72c9c864e5ec23 Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Tue, 6 Jan 2015 20:53:28 +0100
Subject: [PATCH 7/7] build: bring back Camera2 into the package

---
 target/product/core.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/product/core.mk b/target/product/core.mk
index 54ced01..9ec8484 100644
--- a/target/product/core.mk
+++ b/target/product/core.mk
@@ -25,6 +25,7 @@ PRODUCT_PACKAGES += \
     Calculator \
     Calendar \
     CalendarProvider \
+    Camera2 \
     CaptivePortalLogin \
     CertInstaller \
     Contacts \
-- 
2.1.0

