From a77fd95b35f913234b9881bbfa669e61fe97ee80 Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Mon, 6 Oct 2014 23:09:11 +0200
Subject: [PATCH] kernel/lge/x3: disable setting security contexts on rootfs
 inodes

---
 security/selinux/hooks.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/security/selinux/hooks.c b/security/selinux/hooks.c
index f4ac898..3aeca7f 100644
--- a/security/selinux/hooks.c
+++ b/security/selinux/hooks.c
@@ -419,10 +419,6 @@ static int sb_finish_set_opts(struct super_block *sb)
 	    sbsec->behavior > ARRAY_SIZE(labeling_behaviors))
 		sbsec->flags &= ~SE_SBLABELSUPP;
 
-	/* Special handling for sysfs. Is genfs but also has setxattr handler*/
-	if (strncmp(sb->s_type->name, "sysfs", sizeof("sysfs")) == 0)
-		sbsec->flags |= SE_SBLABELSUPP;
-
 	/* Initialize the root inode. */
 	rc = inode_doinit_with_dentry(root_inode, root);
 
-- 
2.1.2
