From 4aae72fc32aeaf06bbd6d581a034f65e0e92dd4b Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Sat, 21 Jun 2014 09:43:15 +0200
Subject: [PATCH] Disable setting security contexts on rootfs inodes

---
 security/selinux/hooks.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/security/selinux/hooks.c b/security/selinux/hooks.c
index 96017ac..cbcb2e3 100644
--- a/security/selinux/hooks.c
+++ b/security/selinux/hooks.c
@@ -411,8 +411,8 @@ static int sb_finish_set_opts(struct super_block *sb)
 	 * Special handling for rootfs. Is genfs but supports
 	 * setting SELinux context on in-core inodes.
 	 */
-	if (strncmp(sb->s_type->name, "rootfs", sizeof("rootfs")) == 0)
-		sbsec->flags |= SE_SBLABELSUPP;
+	// if (strncmp(sb->s_type->name, "rootfs", sizeof("rootfs")) == 0)
+	// 	sbsec->flags |= SE_SBLABELSUPP;
 
 	/* Initialize the root inode. */
 	rc = inode_doinit_with_dentry(root_inode, root);
-- 
1.8.3.2

