From fce515a055005b8db9f4e8587b6c39dae7c304b3 Mon Sep 17 00:00:00 2001
From: streambinder <davidepucci@hiddenhost.org>
Date: Fri, 3 Oct 2014 13:11:48 +0200
Subject: [PATCH] kernel/lge/x3: disable setting security contexts on rootfs
 inodes

---
 security/selinux/hooks.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/security/selinux/hooks.c b/security/selinux/hooks.c
index 9f22cf8..edfffaf 100644
--- a/security/selinux/hooks.c
+++ b/security/selinux/hooks.c
@@ -401,10 +401,6 @@ static int sb_finish_set_opts(struct super_block *sb)
 	    sbsec->behavior > ARRAY_SIZE(labeling_behaviors))
 		sbsec->flags &= ~SE_SBLABELSUPP;
 
-	/* Special handling for sysfs. Is genfs but also has setxattr handler*/
-	if (strncmp(sb->s_type->name, "sysfs", sizeof("sysfs")) == 0)
-		sbsec->flags |= SE_SBLABELSUPP;
-
 	/* Initialize the root inode. */
 	rc = inode_doinit_with_dentry(root_inode, root);
 
-- 
2.1.2
